.github/workflows/phylogenetic-ncbi.yaml

```diff
@@ -95,10 +117,11 @@ jobs:
           --memory 28800mib \
           . \
             deploy_all \
-            --configfile config/h5n1-cattle-outbreak.yaml \
+            --configfile config/${CONFIG_NAME}.yaml \
             --config "${config[@]}"
 
       env: |
```

Comment:
For my own understanding, why is `env` here a string whereas in the docs (and the `summary` job above) it's a map?

Reply:
Here, `env` is a `with` parameter passed to the called workflow. Those are always strings. The called workflow specifies that its `env` parameter will be interpreted as YAML, so the literal block indicator (`|`) allows familiar YAML syntax. (This is [documented](https://github.com/nextstrain/.github/blob/78be1373d33ca9daa439cc5f1641309f811ebe0e/.github/workflows/pathogen-repo-build.yaml.in#L93-L113) by the called workflow.)

Reply:
The `summary` job is using the GH Action [jobs.<job_id>.env](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idenv). Here, it's an [input](https://github.com/nextstrain/.github/blob/78be1373d33ca9daa439cc5f1641309f811ebe0e/.github/workflows/pathogen-repo-build.yaml#L64) to the reusable workflow.

Reply:
(Ah, Tom already answered 😅)

Reply:
Gotcha. Thanks both. The reuse of names seems likely to cause confusion like this - confusion which presumably this improves with more usage of the workflows but is there nonetheless for casual users.